src/ColorPairFactory.cpp

```diff
@@ -0,0 +1,9 @@
+#include "ColorPairFactory.h"
+
+namespace TelCoColorCoder {
+
+    std::unique_ptr<ColorPair> ColorPairFactory::createColorPair(MajorColor major, MinorColor minor) {
```

Comment:
One more option of organizing files - can make this function `inline`, so we can have it only in the .h file and don't need the cpp (to avoid updating the function signature in two places when any changes happen)